package/boot/uboot-envtools/files/qualcommax_ipq60xx

```diff
@@ -20,6 +20,10 @@ netgear,wax214)
 	[ -n "$idx" ] && \
 		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x40000" "0x20000" "2"
 	;;
+tplink,eap610-outdoor)
+	idx="$(find_mtd_index 0:appsblenv)"
+	[ -n "$idx" ] && \
+		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x40000" "0x20000"
```

Comment:
This is the same thing as WAX214 uses, you just did not specify the number of blocks.

Reply:
The number of blocks is redundant, so why specify them?